src/zeit/content/article/article.py

```diff
@@ -193,6 +193,21 @@ def disallowCommentsIfCommentsAreNotShown(object, event):
         object.commentsAllowed = False
 
 
+@grok.subscribe(
+    zeit.content.article.interfaces.IArticle,
+    zope.lifecycleevent.IObjectModifiedEvent)
+def disable_is_amp_if_access_is_restricted(article, event):
+    """Restricted content should not be promoted by Google.
+
+    We prefer an adapter that changes the XML over a property which overrides
+    `is_amp` to return the appropriate state on read, since clients like mobile
+    only read XML.
+
+    """
```

Comment:
Wir wollten inhaltlichen & technischen Grund dokumentieren. Der technische Grund klingt aber eigentlich nach einer allgemeinen Architektur-Entscheidung. Soll die lieber in die Doku @wosc?